sdk/azcore/core.go

```diff
@@ -73,6 +73,11 @@ type ClientOptions = policy.ClientOptions
 type Client struct {
 	pl runtime.Pipeline
 	tr tracing.Tracer
+
+	// cached on the client to support shallow copying with new values
+	tp tracing.Provider
+	mv string
```

Comment:
I can tell that pl is a Pipeline and tr is a Tracer but I can't tell what mv and ns are for. I suggest descriptive names or at least comments.

Reply:
Fixed up the names.